ops/ansible/roles/legislation_explorer/defaults/main.yml

```
@@ -0,0 +1,33 @@
+legislation_explorer_api_url: http://openfisca-api.example
```

Let's use the default value of the local API.
```suggestion
legislation_explorer_api_url: http://localhost:8000
```